<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Receiver</class>
 <widget class="QMainWindow" name="Receiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Receiver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>false</bool>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_transmitter">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Receiver</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_pid">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="text">
           <string>PID:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="output_box">
          <property name="documentTitle">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_clear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QChartView" name="chart_bitstream"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QChartView" name="chart_signal"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionFrequency"/>
    <addaction name="actionSignal_Resolution"/>
   </widget>
   <widget class="QMenu" name="menuModulation">
    <property name="title">
     <string>Modulation</string>
    </property>
    <addaction name="actionNRZ_Polar"/>
    <addaction name="actionManchester"/>
    <addaction name="actionBipolar"/>
   </widget>
   <widget class="QMenu" name="menuEncoding">
    <property name="title">
     <string>Encoding</string>
    </property>
    <addaction name="actionByte_Counting"/>
    <addaction name="actionBite_Insertion"/>
    <addaction name="separator"/>
    <addaction name="actionParity_Bit"/>
    <addaction name="actionCRC"/>
    <addaction name="separator"/>
    <addaction name="actionHamming"/>
   </widget>
   <addaction name="menuSimulation"/>
   <addaction name="menuModulation"/>
   <addaction name="menuEncoding"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFrequency">
   <property name="text">
    <string>&amp;Frequency</string>
   </property>
  </action>
  <action name="actionSignal_Resolution">
   <property name="text">
    <string>&amp;Signal Resolution</string>
   </property>
  </action>
  <action name="actionNRZ_Polar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;NRZ-Polar</string>
   </property>
  </action>
  <action name="actionManchester">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Manchester</string>
   </property>
  </action>
  <action name="actionBipolar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Bipolar</string>
   </property>
  </action>
  <action name="actionASK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;ASK</string>
   </property>
  </action>
  <action name="actionFSK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;FSK</string>
   </property>
  </action>
  <action name="action8_QAM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;8-QAM</string>
   </property>
  </action>
  <action name="actionByte_Counting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Byte Counting</string>
   </property>
  </action>
  <action name="actionBite_Insertion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bite &amp;Insertion</string>
   </property>
  </action>
  <action name="actionParity_Bit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Parity Bit</string>
   </property>
  </action>
  <action name="actionCRC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;CRC</string>
   </property>
  </action>
  <action name="actionHamming">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hamming</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
